config/jobs/kubernetes/sig-release/promo-tools/promo-tools.yaml

```diff
@@ -88,11 +88,11 @@ presubmits:
         - files
         - --manifests=/home/prow/go/src/github.com/kubernetes/k8s.io/artifacts/
         - --dry-run=true
-  kubernetes-sigs/k8s-container-image-promoter:
+  kubernetes-sigs/promo-tools:
```

Comment:
For followup: I would prefer to see this live in config/jobs/kubernetes-sigs/promo-tools

Reply:
> For followup: I would prefer to see this live in config/jobs/kubernetes-sigs/promo-tools

Addressed in https://github.com/kubernetes/test-infra/pull/23626.